common_types/src/datum.rs

```diff
@@ -552,6 +603,19 @@ impl Datum {
             Datum::Int16(v) => v.to_string(),
             Datum::Int8(v) => v.to_string(),
             Datum::Boolean(v) => v.to_string(),
+            Datum::Date(v) => format!(
+                "{:?}",
+                NaiveDate::from_num_days_from_ce_opt((*v) + EPOCH_DAYS_FROM_CE)
+                    .unwrap()
+                    .to_string()
+            ),
```

Comment:
format!  may be not needed here.

Reply:
I will remove the `format`, and rerun the unit tests.

Reply:
> format! may be not needed here.

Yes, it works well without `format`.